LOAN/LoansTemporal/LoanApplicationsTemporal.rdf

```diff
@@ -123,32 +172,26 @@
 		<rdfs:subClassOf rdf:resource="&fibo-fnd-arr-doc;Document"/>
 		<rdfs:subClassOf>
 			<owl:Restriction>
-				<owl:onProperty rdf:resource="&fibo-loan-tem-app;refersToValueOf"/>
-				<owl:someValuesFrom rdf:resource="&fibo-fbc-dae-crt;CreditRating"/>
-			</owl:Restriction>
-		</rdfs:subClassOf>
-		<rdfs:subClassOf>
-			<owl:Restriction>
-				<owl:onProperty rdf:resource="&fibo-loan-tem-bor;appraisedBy"/>
-				<owl:someValuesFrom rdf:resource="&fibo-loan-tem-app;LoanAppraiser"/>
+				<owl:onProperty rdf:resource="&fibo-fnd-arr-doc;isAbout"/>
+				<owl:someValuesFrom rdf:resource="&fibo-fbc-dae-dbt;Borrower"/>
```

Comment:
They're not a Borrower yet, just an Applicant.

Reply:
Probably - but my only goal here was to merge two ontologies and eliminate circularity - did not review in detail, which we can do with SME assistance.